SYCL/Plugin/level_zero_batch_event_status.cpp

```diff
@@ -1,5 +1,5 @@
 // See https://github.com/intel/llvm-test-suite/issues/906
```

Comment:
Eventually this link should go away somehow

Reply:
I added a comment on the issues page. Thanks